src/lib/components/Navbar.svelte

```diff
@@ -45,7 +50,8 @@
     },
     {
       href: 'https://mermaidchart.com',
-      img: './mermaidchart-logo.svg'
+      img: './mermaidchart-logo.svg',
+      isHidden: !isEnabledMermaidChartLinks
     }
```

Comment:
I think it's better to push if `isEnabledMermaidChartLinks`. Then we can avoid the condition in the template, and keep the filtering in one place. 

Another option is to do a `[ ... ].filter((link) => !link.isHidden)`, which would help us have multiple hidden links in different places in the list. 



```
{
      href: 'https://mermaidchart.com',
      img: './mermaidchart-logo.svg'
}
```